### PR TITLE
Remove squircle

### LINKED ISSUE #17: removeSquircle() function?
**Is your feature request related to a problem? Please describe.**
If we're reusing a view, for example in a table view cell, perhaps we'd like to be able to remove the squircle as easily as we can add one?

**Describe the solution you'd like**
I'd like to be able to call view.removeSquircle()

### PR DESCRIPTION
**Pull request by claesjacobsson**
- Adds function to remove squircle and/or border.
- Updated example to illustrate the remove border function.

Fixes #17

### CODE REVIEW

File: Squircle/Squircle.swift [hunk 1]
```diff
@@ -26,8 +26,24 @@ extension UIView {
...
+    
+    /**
+     Remove squircle border.
+     */
+    public func removeSquircleBorder() {
+        self.layer.removeSquircleBorder()
+    }
 }
 
+let squircleBorderName = "squircleBorder"
```
Comment (neobeppe) [inline Squircle/Squircle.swift]: Can this variable be internal? We don't want to expose it to the final user
```suggestion
internal let squircleBorderName = "squircleBorder"
```

Comment (claesjacobsson) [reply Squircle/Squircle.swift]: Makes perfect sense, but my test makes use of it. Use the string in the test instead?

Comment (neobeppe) [reply Squircle/Squircle.swift]: If the test file is importing target with `@testable import` it will have access to internal var and methods too

**Comment by claesjacobsson** [comment]
Ah sorry. Let’s do that 😀 Will you do the change?

**Comment by neobeppe** [comment]
You have just to apply my suggestion in my first comment

**Pull request merged**